microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -519,6 +519,45 @@
         });
     }
 
+    function updateDatasetDetails(state){
+        $.ajax(
+            {
+                url: state.public_endpoint + "/dataset/" + state.dataset_input.value,
+                success: function(data)
+                {
+                    var info = data[state.dataset_input.value]
+                    if (info == null)
+                        return
+                    $("#dataset_title").text(info.title).removeClass("spinner-grow spinner-grow-sm");
+                    if (info.datatype == "WGS"){
+                        $("#dataset_datatype").text("whole genome sequencing, a technique that produces a large number of small DNA sequences from all of the microbial genomes in your sample. With enough sequences you can even see the variation that exists between microbes of the same species. ").removeClass("spinner-grow spinner-grow-sm");
+                    }
+                    else if (info.datatype == "16S"){
+                        $("#dataset_datatype").text('16S.  This technique produces DNA sequences from a specific variable region ("V4") within a microbial gene ("the 16S small subunit ribosomal gene"). DNA sequences from this region of this gene can be used like a microbial barcode, providing researchers evidence of the types of microbes that may be present in your sample.').removeClass("spinner-grow spinner-grow-sm");
+                    }
+                    else {
+                        $("#dataset_datatype").text(info.datatype).removeClass("spinner-grow spinner-grow-sm");
+
+                    }
+                    for (var qiita_id of info['qiita-study-ids'])
+                        $("#dataset_links").append('<li><a href="https://qiita.ucsd.edu/study/description/10317">Qiita Study: ' + qiita_id + '</li>').removeClass("spinner-grow spinner-grow-sm");
```

Comment:
It looks like these are all going to 10317, can the links be specific to the study?

Reply:
@antgonza, is there a public/anonymous API on qiita to query for public study details? I can find these [docs](https://qiita.ucsd.edu/static/doc/html/dev/rest.html?highlight=rest) but it looks like authentication is required. We're looking for a simple way to provide information like the study title and associated publications

Reply:
This could work: https://qiita.ucsd.edu/static/doc/html/downloading.html#access-a-public-study-or-artifact-without-login

Reply:
Like a series of `iframe`s or something? @dhakim87 what do you think?

Reply:
Updated link to point to the public study page from antonio's link.

Reply:
Thanks!!